### PR TITLE
V0.7.1 and get the gce build happening inside Dapper too

### PR DESCRIPTION
**Pull request by SvenDowideit**
@cloudnautique @ibuildthecloud 

not quite done yet, but getting closer here too.

### CODE REVIEW

File: gce/gce-qemu.json [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+ },
  "builders":
  [
    {
      "type": "qemu",
      "name": "qemu-googlecompute",
-     "iso_url": "https://github.com/rancherio/os/releases/download/v0.7.0/rancheros.iso",
-     "iso_checksum": "00eb6b35597ba1904e57e9bb1268eacf",
+     "iso_url": "/rancheros.iso",
+     "iso_checksum": "eb58ac4958cf66bb9c2675cd7e565419",
```
Comment (ibuildthecloud) [inline gce/gce-qemu.json]: ewww.  With each release will you have to update this checksum in this file?

File: scripts/container-install [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 set -x -e
 
-VERSION=${VERSION:?"VERSION is not set"}
+VERSION=${VERSION:?"VERSION is not set SVSVSV"}
```
Comment (ibuildthecloud) [inline scripts/container-install]: SVSVSV??

File: gce/gce-qemu.json (hunk 1 above)
Comment (SvenDowideit) [reply gce/gce-qemu.json]: oh god no, i don't.

File: scripts/container-install (hunk 2 above)
Comment (SvenDowideit) [reply scripts/container-install]: i knows nozink!

**Comment by SvenDowideit** [comment]
@ibuildthecloud updated to use the checksum file on GH

**Pull request merged**